core/templates/pages/splash-page/splash-page.component.ts

```diff
@@ -51,7 +51,7 @@ angular.module('oppia').component('splashPage', {
       ctrl.onClickBrowseLibraryButton = function() {
         SiteAnalyticsService.registerClickBrowseLibraryButtonEvent();
         $timeout(function() {
-          WindowRef.nativeWindow.location = '/community-library';
+          WindowRef.nativeWindow.location = '/learn/math';
```

Comment:
Should we change the function name as per these changes? 

Also, the analytics is still assuming it to be a browse library button click, do we need to change that?

Reply:
Yes let's change to "BrowseLessonsButton"

Reply:
Done.